README.md

```
@@ -75,6 +75,29 @@ and before you ask a question,
 
 If this isn't passing, is there something you can do to help?
 
+## Repository Synchronization with Template
+
+We have integrated an automated repository synchronization feature using the [actions-template-sync](https://github.com/marketplace/actions/actions-template-sync) GitHub Action. This allows the repository to stay in sync with a designated template repository.
```

Markdownlint

Expected: 100; Actual: 259
Line length

(MD013, line-length)